### ISSUE TITLE
Fix output of `make lint`

### ISSUE THREAD

**Issue by joe-bowman** [ingenuity-build/quicksilver#38]
Running `make lint` returns the following errors:
```
x/interchainquery/keeper/msg_server.go:30:22    errcheck  Error return value of `k.SetDatapointForId` is not checked
x/interchainquery/keeper/msg_server.go:36:23    errcheck  Error return value of `k.SetDatapointForId` is not checked
x/interchainstaking/keeper/abci.go:134:16       errcheck  Error return value of `k.Delegate` is not checked
x/interchainstaking/keeper/hooks.go:19:32       errcheck  Error return value of `k.WithdrawDelegationRewards` is not checked
x/interchainstaking/keeper/msg_server.go:38:24  errcheck  Error return value of `k.SetConnectionForPort` is not checked
x/interchainstaking/keeper/msg_server.go:47:25  errcheck  Error return value of `k.SetConnectionForPort` is not checked
x/interchainstaking/keeper/receipt.go:98:14     errcheck  Error return value of `k.MintQAsset` is not checked
x/interchainstaking/keeper/receipt.go:99:22     errcheck  Error return value of `k.TransferToDelegate` is not checked
app/tps_counter_test.go:26:14                   errcheck  Error return value of `tpc.start` is not checked
```

**Comment by levicook** [comment]
my draft pull request cleans up all `errcheck` complaints -- returns the error where it's compatible to; logs it when it can't.

**Comment by levicook** [comment]
There's one error left on the pull request that appears to run a bit deeper; looks like fixing this requires a package upgrade that isn't fully compatible. Not sure on the ripple here & thinking it probably belongs in a different pull request, so just going to submit the PR with this still failing:

```
make lint
golangci-lint run --out-format=tab
x/interchainstaking/keeper/ibc_handlers.go:6:2  staticcheck  SA1019: package github.com/golang/protobuf/proto is deprecated: Use the "google.golang.org/protobuf/proto" package instead. 
make: *** [lint] Error 1
```

**Comment by levicook** [comment]
Should be able to close this after that last PR is merged.

**Comment by joe-bowman** [comment]
closed by #51 #53
